### PR TITLE
Update readme and update changelog

### PR DESCRIPTION
**Pull request by gnufied**

### CODE REVIEW

**Comment by gnufied** [comment]
cc @msau42

File: CHANGELOG-0.1.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# v0.1.0
+
+## Initial release Changelog
+
+* Add a external resize controller which monitors Persistent volume claims and performs
```
Comment (msau42) [inline CHANGELOG-0.1.0.md]: Link to the PR # here.

I want to move the things that were under "Major changes" in previous release notes, to this section

File: CHANGELOG-0.1.0.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# v0.1.0
+
```
Comment (msau42) [inline CHANGELOG-0.1.0.md]: Maybe also add "alpha" at the top

Comment (gnufied) [reply CHANGELOG-0.1.0.md]: fixed

File: CHANGELOG-0.1.0.md (hunk 1 above)
Comment (gnufied) [reply CHANGELOG-0.1.0.md]: I have linked PRs. Since there aren't any previous release notes here - we can start with this list we have?

**Comment by gnufied** [comment]
@msau42 ptal

File: CHANGELOG-0.1.0.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+## Support Status
+
+Alpha
+
+## Initial release Changelog
+
+* [#1](https://github.com/kubernetes-csi/external-resizer/pull/1) Add a external resize controller which monitors Persistent volume claims and performs CSI `ControllerExpandVolume` as needed.
+* [#26](https://github.com/kubernetes-csi/external-resizer/pull/26) If plugin does not implement `ControllerExpandVolume` it performs a no-op expansion and updates PV object.
```
Comment (msau42) [inline CHANGELOG-0.1.0.md]: Can you add the leader election PR here too

Comment (gnufied) [reply CHANGELOG-0.1.0.md]: done

**Pull request merged**